api/app/src/Repository/StagingDeputyshipRepository.php

```diff
@@ -17,4 +19,35 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, StagingDeputyship::class);
     }
+
+    public function findAllPaged(int $pageSize = 1000): \Generator
```

Comment:
This might be refactored out into a helper at some point, to provide other parts of the ingest process with a means to process large database tables a page at a time.